README.md

```diff
@@ -50,72 +50,92 @@ Want to make OneBusAway Alexa better?  We welcome collaboration!  See our [Contr
+
+1. From the [AWS Console > Lambda](https://console.aws.amazon.com/lambda), open the newly created Lambda function and add a _Trigger_ (Event Source) of type `Alexa Skills Kit`.
+1. Note the ARN of the Lambda function at the top right corner of the screen. You will use it to set up a new Alexa Skill in the next section.
+
+See the [lambda maven plugin homepage](https://github.com/SeanRoy/lambda-maven-plugin) for more information on deploying.
+
+### 4. Set up Alexa Skill
+
+1. Go to the [Amazon Developer Console > Alexa](http://developer.amazon.com/edw/home.html)
+1. Add a new skill.  Set _Skill Type_ to `Custom Interaction Model`, set _Invocation Name_ to "one bus away". _Name_ can be anything since this is your development version.
+1. Paste the contents of file `interaction model/schema.json` into "Intent Schema" text box.
+1. Under "Custom Slot Types," click on "Add Slot Type".  Under "Enter type", add `TRANSIT_MODES`.  Under "Enter values", paste the contents of the file `interaction model/customSlotTypes/TRANSIT_MODES`.
 1. Under "Custom Slot Types," click on "Add Slot Type".  Under "Enter type", add `AMAZON.US_CITY`.  Under "Enter values", paste the contents of the file `interaction model/customSlotTypes/AMAZON.US_CITY`.
 1. Into "Sample Utterances" text box, paste the contents of file `interaction model/utterances.txt`.
-   Go Next.  That creates the skill.
-1. Switch back to "Skill Information" section, the first section of wizard.
-1. Note the _Application Id_.  Copy that into `src/main/resources/onebusaway.properties`
-   under the `skill-app-id-development=amzn1...` entry.
-1. Now recompile and re-deploy to Lambda!
-
-CAUTION: Every time you re-deploy to Lambda using the `lambduh-maven-plugin`, you must
-manually re-add "Alexa Skills Kit" as the function's _Event Source_.  You do not need to
-do this if you deploy your code through the Lambda UI in AWS Console.
+1. On Configuration page set up your endpoint by plugging in your Lambda function's ARN.
+   Go Next.  That creates the skill, however it is not functional yet.
+1. At the top of the screen note application _ID_. You will use it to configure Lambda code.
+
+### 5. Configure, rebuild and redeploy Lambda function
+1. Create `src/main/resources/onebusaway.properties` with the following parameters:
+
+```
+skill-app-id-development=
+aws.key-id=
+aws.secret-key=
+googlemaps.api-key=
+onebusaway.api-key=
+```
+
+Fill in the values:
+
+`skill-app-id-development` is Alexa Skill ID from previous step
+
+`aws.key-id` and `aws.secret-key` are `{appExecutionAccessKey}` and `{appExecutionSecretKey}` respectively as generated by CloudFormation
+
+`googlemaps.api-key` Google Maps GeoCoding API Key which can be obtained from [Google Developers](https://developers.google.com/maps/get-started/)
+
+`onebusaway.api-key` can be obtained from your local OBA region. Typically, you can use `TEST` while you waiting for a key.

```

Comment:
Typo - Should be "Typically, you can use `TEST` while you **are** waiting for a key.**